Build.ps1

```diff
@@ -1,85 +1,15 @@
-param(
-    [String] $majorMinor = "0.0",  # 2.0
-    [String] $patch = "0",         # $env:APPVEYOR_BUILD_VERSION
-    [String] $customLogger = "",   # C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
-    [Switch] $notouch,
-    [String] $sln                  # e.g serilog-sink-name
-)
+Push-Location $PSScriptRoot
```

Comment:
The existence check would be fine; where possible I've tried adding empty test projects in these cases, which is probably the ideal option if it's not too much trouble.